footer component added

Pushpa88431 opened the pull request.

mohsinrony commented:
Hi Pushpa

Have we added the image(logo)/imageUrl into the assets folder?

Because, when I pulled the code for the header, it doesn't show me any image.

Pushpa88431 commented:
Hi,
That is only for the test. Also we might can add image in future thats why
i added it, otherwise we can remove initially and add later if we want.

On Tue, Feb 20, 2024, 6:50 AM Md Mohsin ***@***.***> wrote:

> Hi Pushpa
>
> Have we added the image(logo)/imageUrl into the assets folder?
>
> Because, when I pulled the code for the header, it doesn't show me any
> image.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mohsinrony/movie-search-app/pull/6#issuecomment-1953484226>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASAAGDWXCIUEYIBBHGJXZLLYUQTQVAVCNFSM6AAAAABDPWEPG2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSNJTGQ4DIMRSGY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
